src/Microsoft.VisualStudio.Editors/SettingsDesigner/AppConfigSerializer.vb

```diff
@@ -402,8 +397,8 @@ Namespace Microsoft.VisualStudio.Editors.SettingsDesigner
                 Dim map As New ExeConfigurationFileMap
                 Try
                     map.ExeConfigFilename = AppConfigDocData.Name
-                    map.LocalUserConfigFilename = ConfigHelperService.GetUserConfigurationPath(hierSp, project, ConfigurationUserLevel.PerUserRoamingAndLocal, UnderVsHost, BuildConfig)
-                    map.RoamingUserConfigFilename = ConfigHelperService.GetUserConfigurationPath(hierSp, project, ConfigurationUserLevel.PerUserRoaming, UnderVsHost, BuildConfig)
+                    map.LocalUserConfigFilename = ConfigHelperService.GetUserConfigurationPath(hierSp, project, ConfigurationUserLevel.PerUserRoamingAndLocal, underHostingProcess:=False, buildConfiguration:=BuildConfig)
```

Comment:
Okay, as far as I can tell, what the ```underHostingProcess``` parameter does is determine which app.config to read (App.exe.config vs. App.vshost.exe.config).  The value is not actually persisted in said config.